lib/core/data/storage_service.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:hive_flutter/adapters.dart';
+import 'package:myflix/core/models/movie_model.dart';
+
+class StorageService {
+  String key = 'movies';
+
+  Future<List<Movie>> fetchWatchList() async {
+    Box<Movie> movieBox = Hive.box<Movie>('watchListBox');
```

Comment:
Can you create a getter for this so you don't repeatedly create the instance in each of your methods where you need `movieBox`?